providers/keycloak_oidc.go

```diff
@@ -21,6 +21,8 @@ func NewKeycloakOIDCProvider(p *ProviderData, opts options.Provider) *KeycloakOI
 		name: keycloakOIDCProviderName,
 	})
 
+	p.BackendLogoutURL = opts.KeycloakConfig.BackendLogoutURL
+
```

Comment:
Final suggestion, move this to `setProviderDefaults(defaults providerDefaults)` in `provider_data.go`

Reply:
done